pre-uninstall.sh

```diff
@@ -29,13 +29,11 @@ while true; do
   shift
 done
 
-# Root check
-if [ "$EUID" -ne 0 ] && [ "$NO_SUDO" = false ]
-  then echo "This program requires root permissions"
+if [ "$EUID" -ne 0 ]
```

Comment:
Did you forget a check for the NO_SUDO option?

Reply:
I missed it in the main branch too. Here is the PR: https://github.com/TamtamHero/fw-fanctrl/pull/79